WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailNewHeaderView.swift

```diff
@@ -461,6 +461,7 @@ fileprivate struct ReaderDetailTagsWrapperView: UIViewRepresentable {
 
         // ensure that the collection view hugs its content.
         view.setContentHuggingPriority(.defaultHigh, for: .vertical)
+        view.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
```

Comment:
This is what fixes the issue you've found in https://github.com/wordpress-mobile/WordPress-iOS/pull/21674#pullrequestreview-1660923880. Somehow the default content compression resistance won over SwiftUI's defaults... I think? 🤷🏼‍♂️